examples/notebooks/datastores/7_cmems_data_store.ipynb

```diff
@@ -104,16 +110,16 @@
          "default": "https://cmems-cas.cls.fr/cas/login",
          "type": "string"
         },
-        "cmems_user": {
-         "description": "Preferably set by environment variable CMEMS_USER",
-         "title": "CMEMS User",
-         "type": "string"
-        },
-        "cmems_user_password": {
+        "cmems_password": {
          "description": "Preferably set by environment variable CMEMS_PASSWORD",
          "title": "CMEMS User Password",
```

Comment:
```suggestion
         "title": "CMEMS Password",
```

Reply:
I checked this again, it works fine for me. Note that environment variables are renamed to CMEMS_USERNAME and CMEMS_PASSWORD.